accounting/bank/choices.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2017-2018 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+Bank choices.
+"""
+
+from django.utils.translation import ugettext_lazy as _
+from djchoices import ChoiceItem, DjangoChoices
+
+
+class BankAccountIdentifiers(DjangoChoices):
+    """
+    Choices for bank account identification.
+
+    This matches what TransferWise works with for bank identification.
+    It should be kept up to date depending on the API version used.
+    """
```

Comment:
We can, but I did not have time to look into / implement a TransferWise API. They don't even have a Python client that I know of so I'd have to write out auth as well.

Reply:
I don't think you need to implement the Transferwise integration now - just to remove all the fields that are specific to transferwise, if at the end the only thing we store for Transferwise recipients is their recipient id. The only accounts for which we would need to store bank information would be the ones not using transferwise, and for these the fields are the international standard fields, rather than all the local bank details transferwise manages:
* Account number (variable length - that's for example the IBAN in Europe)
* BIC/SWIFT

And that's it! :) The bank only need these two, plus the name & address of recipient and its bank, to be able to do transfers anywhere.

Reply:
Alright, this sounds good to me. :+1: